lib/MFA.php

```diff
@@ -138,4 +137,38 @@ public function verifyFactor(
 
         return Resource\VerificationChallenge::constructFromResponse($response);
     }
+
+
+    /**
+     * Verifies the one time password provided by the end-user.
+     *
+     * @param string $authenticationChallengeId - The ID of the authentication challenge that provided the user the verification code.
+     * @param string $code - The verification code sent to and provided by the end user.
+    */
+
+    public function verifyChallenge(
```

Comment:
Could we rewrite `verifyFactor` above so that it calls `verifyChallenge` for its implementation?

We still want to use the new endpoints even for anyone still using the old method.

Reply:
Thanks Marshall! Took a look at the Python SDK and followed suit from that :-)